feat(http): Replace usage of "payload" with "content"

bsmth opened the pull request:
### Description

This is a maintenance PR part of work to refresh HTTP docs.

Fixes #20741 which looks like a simple change, but it's trickier because we have to be more abstract to encompass HTTP/1.1 -> HTTP/3 concepts.

__Changes:__
* `Payload_body` -> `HTTP_Content`
* `Payload_header` -> `Content_header`
* Contextual update to:
  * Transfer encoding
  * Content encoding
  * 413 status err
* Other instances of "payload" -> "content" where it makes sense

### Motivation

Keeping content up-to-date and useful.


__Related issues and pull requests:__

- [x] https://github.com/mdn/content/pull/20743

bsmth committed:
feat(http): Update usage of 'payload' when we're talking about request content

bsmth committed:
feat(http): Update usage of 'payload' when we're talking about request content

bsmth committed:
feat(http): Update usage of 'payload' when we're talking about HTTP content

bsmth committed:
feat(http): Update usage of 'payload' when we're talking about HTTP content

bsmth committed:
feat(http): Update usage of 'payload' when we're talking about HTTP content

bsmth committed:
feat(http): Update usage of 'payload' when we're talking about HTTP content

bsmth committed:
Merge branch 'main' into 20741-payload-content

bsmth committed:
feat(http): Update usage of 'payload' when we're talking about HTTP content

LostBeard committed:
Fixed instanceof summary to better match instanceof page (#35062)

* Fixed instanceof summary to better match instance of page

* Update files/en-us/web/javascript/guide/expressions_and_operators/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

Josh-Cena committed:
Fix code-link containing spaces, part 1 (#35065)

dependabot[bot] committed:
Bump husky from 9.1.1 to 9.1.2 (#35061)

Bumps [husky](https://github.com/typicode/husky) from 9.1.1 to 9.1.2.
- [Release notes](https://github.com/typicode/husky/releases)
- [Commits](https://github.com/typicode/husky/compare/v9.1.1...v9.1.2)

---
updated-dependencies:
- dependency-name: husky
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

ankineri committed:
Fix a typo for cssText (#35073)

Josh-Cena committed:
Fix code-links containing spaces in CSS/JS (#35075)

queengooborg committed:
Convert noteblocks for learn/forms folder (#35071)

* Convert noteblocks for learn/forms folder

This PR converts the noteblocks for the 'learn/forms' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fix formatting

tasinttttttt committed:
Updates api/element/Element:QuerySelectorAll() example, fixes #35003 (#35006)

* Updates api/element/Element:QuerySelectorAll() example, fixes #35003

* feat: remove unused example

* Update index.md

* Update files/en-us/web/api/element/queryselectorall/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for learn/html folder (#35072)

* Convert noteblocks for learn/html folder

This PR converts the noteblocks for the 'learn/html' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fix formatting

* Update files/en-us/learn/html/introduction_to_html/debugging_html/index.md

---------

Co-authored-by: Estelle Weyl <estelle@weyl.org>

queengooborg committed:
Convert callout noteblocks to GFM syntax (#35066)

queengooborg committed:
Convert noteblocks for learn/common_questions folder (#35069)

This PR converts the noteblocks for the 'learn/common_questions' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

queengooborg committed:
Convert noteblocks for learn/server-side folder (#35077)

* Convert noteblocks for learn/server-side folder

This PR converts the noteblocks for the 'learn/server-side' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Update files/en-us/learn/server-side/first_steps/introduction/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for learn/css folder (#35070)

* Convert noteblocks for learn/css folder

This PR converts the noteblocks for the 'learn/css' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fix formatting

* Update files/en-us/learn/css/building_blocks/backgrounds_and_borders/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* Update files/en-us/learn/css/css_layout/legacy_layout_methods/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for learn/tools_and_testing folder (#35078)

* Convert noteblocks for learn/tools_and_testing folder

This PR converts the noteblocks for the 'learn/tools_and_testing' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Apply suggestions from code review

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* Update files/en-us/learn/tools_and_testing/cross_browser_testing/accessibility/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

gagan-bhullar-tech committed:
fix: change word lossy (#35090)

improper word lossy

gagan-bhullar-tech committed:
import destructure default export (#35104)

import destructure fix

queengooborg committed:
Convert noteblocks for web/svg folder (#35102)

This PR converts the noteblocks for the 'web/svg' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

queengooborg committed:
Convert noteblocks for web folder (#35105)

* Convert noteblocks for web folder

This PR converts the noteblocks for the 'web' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Revert problematic notes

queengooborg committed:
Convert noteblocks for webassembly folder (#35103)

* Convert noteblocks for webassembly folder

This PR converts the noteblocks for the 'webassembly' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Revert problematic noteblocks

* Update files/en-us/webassembly/using_the_javascript_api/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Revert conversion to GFM noteblocks for notes with formatting bugs (#35106)

* Revert conversion to GFM noteblocks for notes with formatting bugs

* A few more?

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/media folder (#35099)

* Convert noteblocks for web/media folder

This PR converts the noteblocks for the 'web/media' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Apply suggestions from code review

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for learn folder (#35079)

* Convert noteblocks for learn folder

This PR converts the noteblocks for the 'learn' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Apply suggestions from code review

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for mozilla/firefox folder (#35081)

This PR converts the noteblocks for the 'mozilla/firefox' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

queengooborg committed:
Convert noteblocks for related folder (#35082)

This PR converts the noteblocks for the 'related' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

queengooborg committed:
Convert noteblocks for web/accessibility folder (#35083)

* Convert noteblocks for web/accessibility folder

This PR converts the noteblocks for the 'web/accessibility' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Revert

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/html/attributes folder (#35084)

* Convert noteblocks for web/html/attributes folder

This PR converts the noteblocks for the 'web/html/attributes' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Update files/en-us/web/html/attributes/rel/preload/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/html/element folder (#35085)

* Convert noteblocks for web/html/element folder

This PR converts the noteblocks for the 'web/html/element' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Revert

* Change?

* Fix

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/html/global_attributes folder (#35086)

* Convert noteblocks for web/html/global_attributes folder

This PR converts the noteblocks for the 'web/html/global_attributes' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Update files/en-us/web/html/global_attributes/is/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/html folder (#35087)

This PR converts the noteblocks for the 'web/html' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

estelle committed:
typo: css comment (#35055)

* css comment

* text to css

queengooborg committed:
Convert noteblocks for learn/javascript folder (#35074)

* Convert noteblocks for learn/javascript folder

This PR converts the noteblocks for the 'learn/javascript' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fix formatting

* Update files/en-us/learn/javascript/first_steps/useful_string_methods/index.md

* Fix

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/http/headers folder (#35088)

* Convert noteblocks for web/http/headers folder

This PR converts the noteblocks for the 'web/http/headers' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fixes

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/http folder (#35089)

* Convert noteblocks for web/http folder

This PR converts the noteblocks for the 'web/http' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fixes

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/javascript/reference/global_objects folder (#35091)

* Convert noteblocks for web/javascript/reference/global_objects folder

This PR converts the noteblocks for the 'web/javascript/reference/global_objects' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fixes

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/javascript/reference/operators folder (#35092)

* Convert noteblocks for web/javascript/reference/operators folder

This PR converts the noteblocks for the 'web/javascript/reference/operators' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fixes

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/javascript/reference/regular_expressions folder (#35093)

* Convert noteblocks for web/javascript/reference/regular_expressions folder

This PR converts the noteblocks for the 'web/javascript/reference/regular_expressions' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fixes

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for mdn folder (#35080)

* Convert noteblocks for mdn folder

This PR converts the noteblocks for the 'mdn' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Revert

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/javascript/reference/statements folder (#35094)

* Convert noteblocks for web/javascript/reference/statements folder

This PR converts the noteblocks for the 'web/javascript/reference/statements' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fixes

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for learn/accessibility folder (#35068)

* Convert noteblocks for learn/accessibility folder

This PR converts the noteblocks for the 'learn/accessibility' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fix formatting

* Apply suggestions from code review

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/javascript/reference folder (#35095)

* Convert noteblocks for web/javascript/reference folder

This PR converts the noteblocks for the 'web/javascript/reference' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fixes

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/javascript folder (#35096)

* Convert noteblocks for web/javascript folder

This PR converts the noteblocks for the 'web/javascript' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fixes

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for glossary folder (#35067)

* Convert noteblocks for glossary folder

This PR converts the noteblocks for the 'glossary' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Apply suggestions from code review

* Fix noteblock

* Update files/en-us/glossary/global_object/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/manifest folder (#35097)

* Convert noteblocks for web/manifest folder

This PR converts the noteblocks for the 'web/manifest' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fix broken noteblocks

* Update files/en-us/web/manifest/orientation/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/mathml folder (#35098)

* Convert noteblocks for web/mathml folder

This PR converts the noteblocks for the 'web/mathml' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fix broken noteblocks

queengooborg committed:
Convert noteblocks for web/svg/attribute folder (#35100)

* Convert noteblocks for web/svg/attribute folder

This PR converts the noteblocks for the 'web/svg/attribute' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fix broken noteblocks

queengooborg committed:
Convert noteblocks for web/svg/element folder (#35101)

* Convert noteblocks for web/svg/element folder

This PR converts the noteblocks for the 'web/svg/element' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Fix broken noteblocks

Josh-Cena committed:
Fix code-links containing spaces, API1 (#35076)

hamishwillee commented on files/en-us/glossary/http_content/index.md:
```diff
@@ -0,0 +1,50 @@
+---
+title: Content
```
Should it be "HTTP Content"?

hamishwillee commented on files/en-us/glossary/content_header/index.md:
```diff
@@ -0,0 +1,27 @@
+
+{{GlossarySidebar}}
+
+**Content headers** is a group of [HTTP headers](/en-US/docs/Web/HTTP/Headers) that describe information related to safe transport and reconstruction of the original resource from one or more messages.
```
```suggestion
**Content headers** are a group of [HTTP headers](/en-US/docs/Web/HTTP/Headers) that describe information related to safe transport and reconstruction of the original resource from one or more messages.
```

hamishwillee commented on files/en-us/glossary/content_header/index.md:
```diff
@@ -0,0 +1,27 @@
+These headers may be present in both HTTP request and response messages and include information like the length of the data, which part of the resource is carried in the data (for a multi-part message), any encoding applied for transport, and message integrity checks.
+The group of content headers are:
+
+- {{HTTPHeader("Content-Length")}}
+- {{HTTPHeader("Content-Range")}}
+- {{HTTPHeader("Trailer")}}
+- {{HTTPHeader("Transfer-Encoding")}}
```
IS this set correct? I mean I haven't checked, but the implication is that this set is complete, so I'm wondering if others I can think of are here also - such as `Content-Type`, `Content-Encoding`, `Content-Language`, `Content-Location` . I would also question whether `Trailer` is a content header? [what does the spec say?]

hamishwillee replied:
Further below you talk about `Representation header` which has overlap - would be good to understand whether we need to cross link/explain the differences. I guess there are some payload (content) headers that are not representation headers, so those are a subset. But we should say.

hamishwillee commented on files/en-us/glossary/http_content/index.md:
```diff
@@ -0,0 +1,50 @@
+
+{{GlossarySidebar}}
+
+In HTTP, **content** describes the 'information part' of data sent after the header section when message framing has been removed (like chunked transfer encoding in HTTP/1.1).
```
```suggestion
In HTTP messages, **content** describes the 'information part' of data sent after the header section when message framing has been removed (like chunked transfer encoding in HTTP/1.1).
```

hamishwillee replied:
Note " (like chunked transfer encoding in HTTP/1.1)." doesn't mean anything to me, so is it helpful?

hamishwillee commented on files/en-us/glossary/http_content/index.md:
```diff
@@ -0,0 +1,50 @@
+
+{{GlossarySidebar}}
+
+In HTTP, **content** describes the 'information part' of data sent after the header section when message framing has been removed (like chunked transfer encoding in HTTP/1.1).
+This was defined as a "payload" in HTTP/1.1, but this has been replaced with "content" to avoid confusion with frame payloads in HTTP/2 and HTTP/3.
```
```suggestion
This part of the message was previously called the "payload" in HTTP/1.1 specifications, but is now referred as "content" in order to avoid confusion with frame payloads in HTTP/2 and HTTP/3.
```

hamishwillee replied:
OK, so just to be clear, my understanding is that an HTTP message consists of a bunch of headers and a body. That body contains data, which is more than just "information" - for example in the transfer chunked thingy you're including chunk information. 

So the payload/content is the actual information that would be used by the recipient with any extra framing etc removed. This include metadata in the `trailer` header but not message framing. Is there anything other than message framing that might have to be removed?

So in your examples below the content is "Mozilla Developer Network"

on that basis maybe

```suggestion
In HTTP messages, the **content** describes the 'information' conveyed in the data sent after the header section ("message body"), after any message framing from HTTP/1.1 chunked transfer encoding has been removed.
This was referred to as the message "payload" in the HTTP/1.1 specification, but is now defined as "content" to avoid confusion with the term "frame payloads" in HTTP/2 and HTTP/3.
```

hamishwillee commented on files/en-us/web/http/headers/content-encoding/index.md:
```diff
@@ -7,13 +7,16 @@ browser-compat: http.headers.Content-Encoding
-The **`Content-Encoding`** {{Glossary("representation header")}} lists any encodings that have been applied to the representation (message payload), and in what order.
-This lets the recipient know how to decode the representation in order to obtain the original payload format.
-Content encoding is mainly used to compress the message data without losing information about the origin media type.
+The **`Content-Encoding`** {{Glossary("representation header")}} lists any encodings that have been applied to a resource, and in what order.
```
I'm OK with removing "the representation" because I'm not sure it helps here. But I think that the useful part here was actually "message payload" because these encodings are applied to the resource as it is sent in the message body, not the resource in abstract, before it is sent. I _assume_ they are applied to the content before any chunked encoding or whatever. So perhaps

```suggestion
The **`Content-Encoding`** {{Glossary("representation header")}} lists any encodings that have been applied to a resource, and in what order.
```


```suggestion
The **`Content-Encoding`** {{Glossary("representation header")}} lists any encodings that have been applied to the message {{glossary("HTTP content,"content"")}}, and in what order.
```

hamishwillee commented on files/en-us/web/http/headers/transfer-encoding/index.md:
```diff
@@ -7,9 +7,10 @@ browser-compat: http.headers.Transfer-Encoding
 {{HTTPSidebar}}
 
-The **`Transfer-Encoding`** header specifies the form of encoding used to safely transfer the {{Glossary("Payload body","payload body")}} to the user.
+The **`Transfer-Encoding`** header specifies the form of encoding used to safely transfer messages between nodes on the network.
```
```suggestion
The **`Transfer-Encoding`** header specifies the form of encoding used to transfer messages between nodes on the network.
```

hamishwillee commented on files/en-us/web/http/headers/transfer-encoding/index.md:
```diff
@@ -7,9 +7,10 @@ browser-compat: http.headers.Transfer-Encoding
 
-> **Note:** [HTTP/2](https://en.wikipedia.org/wiki/HTTP/2) disallows all uses of the Transfer-Encoding header other than the HTTP/2 specific: `"trailers"`. HTTP 2 provides its own more efficient mechanisms for data streaming than chunked transfer and forbids the use of the header. Usage of the header in HTTP/2 may likely result in a specific `protocol error` as HTTP/2 Protocol prohibits the use.
+> **Warning:** [HTTP/2](https://en.wikipedia.org/wiki/HTTP/2) disallows all uses of the `Transfer-Encoding` header other than the HTTP/2 specific: `"trailers"`.
+> HTTP/2 provides more efficient mechanisms for data streaming than chunked transfer and forbids the use of the header. Usage of the header in HTTP/2 may likely result in a specific `protocol error` as HTTP/2 Protocol prohibits the use.
```
Should it be HTTP/2 and later ... ?

hamishwillee commented on files/en-us/web/http/messages/index.md:
```diff
@@ -19,7 +19,7 @@ HTTP requests, and responses, share similar structure and are composed of:
 4. An optional _body_ containing data associated with the request (like content of an HTML form), or the document associated with a response. The presence of the body and its size is specified by the start-line and HTTP headers.
 
-The start-line and HTTP headers of the HTTP message are collectively known as the _head_ of the requests, whereas its payload is known as the _body_.
+The start-line and HTTP headers of the HTTP message are collectively known as the _head_ of the requests, whereas its content is known as the _body_.
```
Well, as we know, content and body are not synonymous, since the payload/content can have message framing.

```suggestion
The start-line and HTTP headers of the HTTP message are collectively known as the _head_ of the requests, and the part afterwards that contains its content is known as the _body_.
```

hamishwillee commented on files/en-us/web/http/messages/index.md:
```diff
@@ -97,7 +97,7 @@ Many different headers can appear in responses. These can be divided into severa
 ### Body
 
-The last part of a response is the body. Not all responses have one: responses with a status code that sufficiently answers the request without the need for corresponding payload (like {{HTTPStatus("201")}} **`Created`** or {{HTTPStatus("204")}} **`No Content`**) usually don't.
+The last part of a response is the body. Not all responses have one: responses with a status code that sufficiently answers the request without the need to include content (like {{HTTPStatus("201", "201 Created")}} or {{HTTPStatus("204", "204 No Content")}}) usually don't.
```
Should content be a glossary link here?

hamishwillee commented on files/en-us/web/http/methods/get/index.md:
```diff
@@ -7,9 +7,11 @@ browser-compat: http.methods.GET
 {{HTTPSidebar}}
 
-The **HTTP `GET` method** requests a representation of the specified resource. Requests using `GET` should only be used to request data (they shouldn't include data).
+The **HTTP `GET` method** requests a representation of the specified resource. Requests using `GET` should only be used to request data and shouldn't include content.
```
Glossary link on first use?

hamishwillee commented on files/en-us/web/http/methods/index.md:
```diff
@@ -16,7 +16,7 @@ HTTP defines a set of **request methods** to indicate the desired action to be p
   - : The `POST` method submits an entity to the specified resource, often causing a change in state or side effects on the server.
 - [`PUT`](/en-US/docs/Web/HTTP/Methods/PUT)
-  - : The `PUT` method replaces all current representations of the target resource with the request payload.
+  - : The `PUT` method replaces all current representations of the target resource with the request content.
```
glossary link on first use. Didn't matter with payload because it was kind of well understood, but now maybe worth linking?

hamishwillee commented on files/en-us/web/http/methods/patch/index.md:
```diff
@@ -76,7 +76,7 @@ Content-Length: 100
 A successful response is indicated by any [2xx](https://httpwg.org/specs/rfc9110.html#status.2xx) status code.
 
-In the example below a {{HTTPStatus("204")}} response code is used, because the response does not carry a payload body. A {{HTTPStatus("200")}} response could have contained a payload body.
+In the example below a {{HTTPStatus("204")}} response code is used, because the response does not contain a body. A {{HTTPStatus("200")}} response that contains a body with information about the action could be used instead, for instance.
```
```suggestion
In the example below a {{HTTPStatus("204")}} response code is used, because the response does not contain a body (alternatively, a {{HTTPStatus("200")}} response that contains a body with information about the action could be used instead).
```

hamishwillee commented on files/en-us/web/http/methods/put/index.md:
```diff
@@ -7,7 +7,7 @@ browser-compat: http.methods.PUT
 {{HTTPSidebar}}
 
-The **HTTP `PUT` request method** creates a new resource or replaces a representation of the target resource with the request payload.
+The **HTTP `PUT` request method** creates a new resource or replaces a representation of the target resource with the request content.
```
Glossary link?

hamishwillee replied:
I'm going to stop saying that, but I think it is worth adding on first-use in doc everywhere.

hamishwillee commented on files/en-us/web/http/status/index.md:
```diff
@@ -134,7 +134,7 @@ The status codes listed below are defined by [RFC 9110](https://httpwg.org/specs
 - {{HTTPStatus(412, "412 Precondition Failed")}}
   - : The client has indicated preconditions in its headers which the server does not meet.
-- {{HTTPStatus(413, "413 Payload Too Large")}}
+- {{HTTPStatus(413, "413 Content Too Large")}}
```
FWIW This one is an oddity. We're defining content to mean information = body not including any message framing. But actually this error probably means "body too large". Maybe :-)

bsmth replied:
Yeah that's true. This is where it's tricky to define across the different protocols

queengooborg committed:
Remove page for battery Permissions-Policy value (#34997)

hamishwillee committed:
FF129 Relnote: Navigator.vibrate() removed (#34864)

* FF129 Relnote: Navigator.vibrate() removed

* Update index.md

hamishwillee committed:
FF129 Release note - CSSPageDescriptors, CSSPageRule.style (#34978)

* FF129 Relnote CSSPageDescriptors, CSSPageRule.style

* Apply suggestions from code review

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Update files/en-us/mozilla/firefox/releases/129/index.md

---------

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

queengooborg committed:
Convert noteblocks for learn folder (#35111)

This PR converts the noteblocks for the 'learn' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

queengooborg committed:
Convert noteblocks for mdn folder (#35112)

This PR converts the noteblocks for the 'mdn' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

capaj committed:
docs: add mention of the android signing gotcha (#34375)

* docs: add mention of the android signing gotcha

as per https://blog.mozilla.org/addons/2023/10/05/changes-to-android-extension-signing/

* Update files/en-us/mozilla/add-ons/webextensions/manifest.json/browser_specific_settings/index.md

Co-authored-by: rebloor <git@sherpa.co.nz>

---------

Co-authored-by: rebloor <git@sherpa.co.nz>

Josh-Cena committed:
Fix code-links containing spaces, API2 (#35109)

queengooborg committed:
Convert noteblocks for webassembly folder (#35116)

This PR converts the noteblocks for the 'webassembly' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

queengooborg committed:
Convert noteblocks for mozilla/firefox folder (#35113)

This PR converts the noteblocks for the 'mozilla/firefox' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

wbamberg committed:
Fix issue 28752, part 3: update querySelector* (#35110)

queengooborg committed:
Convert noteblocks for web/html folder (#35114)

* Convert noteblocks for web/html folder

This PR converts the noteblocks for the 'web/html' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

* Update files/en-us/web/html/element/a/index.md

* Apply suggestions from code review

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/http folder (#35115)

This PR converts the noteblocks for the 'web/http' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

queengooborg committed:
Convert noteblocks for web folder (#35117)

This PR converts the noteblocks for the 'web' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

queengooborg committed:
Convert noteblocks for mozilla/add-ons/webextensions/api folder (part 1) (#35118)

Convert noteblocks for mozilla/add-ons/webextensions/api folder

This PR converts the noteblocks for the 'mozilla/add-ons/webextensions/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 1.

queengooborg committed:
Convert noteblocks for mozilla/add-ons/webextensions/api folder (part 2) (#35119)

* Convert noteblocks for mozilla/add-ons/webextensions/api folder

This PR converts the noteblocks for the 'mozilla/add-ons/webextensions/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 2.

* Update files/en-us/mozilla/add-ons/webextensions/api/browseraction/onclicked/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/api folder (part 9) (#35133)

queengooborg committed:
Convert noteblocks for web/api folder (part 6) (#35130)

queengooborg committed:
Convert noteblocks for web/api folder (part 4) (#35128)

queengooborg committed:
Convert noteblocks for web/api folder (part 2) (#35126)

queengooborg committed:
Convert noteblocks for mozilla/add-ons/webextensions/api folder (part 3) (#35120)

* Convert noteblocks for mozilla/add-ons/webextensions/api folder

This PR converts the noteblocks for the 'mozilla/add-ons/webextensions/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 3.

* Update files/en-us/mozilla/add-ons/webextensions/api/clipboard/index.md

* Update files/en-us/mozilla/add-ons/webextensions/api/clipboard/index.md

* Update files/en-us/mozilla/add-ons/webextensions/api/commands/index.md

* Update files/en-us/mozilla/add-ons/webextensions/api/clipboard/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for mozilla/add-ons/webextensions/api folder (part 4) (#35121)

* Convert noteblocks for mozilla/add-ons/webextensions/api folder (part 4)

This PR converts the noteblocks for the 'mozilla/add-ons/webextensions/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 4.

* Apply suggestions from code review

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for mozilla/add-ons/webextensions/api folder (part 5) (#35122)

* Convert noteblocks for mozilla/add-ons/webextensions/api folder (part 5)

This PR converts the noteblocks for the 'mozilla/add-ons/webextensions/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 5.

* Update files/en-us/mozilla/add-ons/webextensions/api/pageaction/onclicked/index.md

* Update files/en-us/mozilla/add-ons/webextensions/api/pageaction/seticon/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for mozilla/add-ons/webextensions/api folder (part 6) (#35123)

* Convert noteblocks for mozilla/add-ons/webextensions/api folder (part 6)

This PR converts the noteblocks for the 'mozilla/add-ons/webextensions/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 6.

* Update files/en-us/mozilla/add-ons/webextensions/api/types/browsersetting/clear/index.md

* Update files/en-us/mozilla/add-ons/webextensions/api/types/browsersetting/get/index.md

* Update files/en-us/mozilla/add-ons/webextensions/api/types/browsersetting/set/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/api folder (part 1) (#35125)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 1.

queengooborg committed:
Convert noteblocks for mozilla/add-ons/webextensions folder (#35124)

* Convert noteblocks for mozilla/add-ons/webextensions folder

This PR converts the noteblocks for the 'mozilla/add-ons/webextensions' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 7.

* Apply suggestions from code review

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

queengooborg committed:
Convert noteblocks for web/api folder (part 3) (#35127)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 3.

queengooborg committed:
Convert noteblocks for web/api folder (part 5) (#35129)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 5.

queengooborg committed:
Convert noteblocks for web/api folder (part 7) (#35131)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 7.

queengooborg committed:
Convert noteblocks for web/api folder (part 8) (#35132)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 8.

queengooborg committed:
Convert noteblocks for web/api folder (part 10) (#35134)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 10.

queengooborg committed:
Convert noteblocks for web/api folder (part 11) (#35135)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 11.

williewillus committed:
window.postMessage: clarify what happens when targetOrigin is not provided (#34981)

OnkarRuikar committed:
fix: convert fence tags from text->plain (#35140)

* fix: convert fence tags from text->plain

* Apply suggestions from code review

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

OnkarRuikar committed:
css(fix): add note to the scripting media feature (#35057)

PassionPenguin committed:
chore: convert noteblocks for `/web/css` (part 4) (#35150)

chores: convert noteblocks

PassionPenguin committed:
chore: convert noteblocks for `/web/css` (part 2) (#35148)

chores: convert noteblocks

Josh-Cena committed:
Remove self links, other pages (#35041)

* Remove self links, other pages

* Fix link

* Fix a broken link

---------

Co-authored-by: Estelle Weyl <estelle@weyl.org>

hamishwillee committed:
ReadableStreamDefaultController.close() - invalid exception reasons (#34690)

Josh-Cena committed:
Fix code-links containing spaces, API3 (#35139)

estelle committed:
minor typo (#35161)

estelle committed:
Minor: Pedantic sentence re animatable properties (#35162)

Pedantic sentence

the page used to list CSS animatable properties, but now all properties are animateable and the linked page is an explainer, so updating sentence / link to remove it "if animatable" connotation.

queengooborg committed:
Convert noteblocks for web/api folder (part 12) (#35153)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 12.

queengooborg committed:
Convert noteblocks for web/api folder (part 13) (#35154)

* Convert noteblocks for web/api folder (part 13)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 13.

* Update files/en-us/web/api/urlpattern/urlpattern/index.md

* Update files/en-us/web/api/urlpattern/urlpattern/index.md

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

* Fix format

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

queengooborg committed:
Convert noteblocks for web/api folder (part 14) (#35155)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 14.

queengooborg committed:
Convert noteblocks for web/api folder (part 15) (#35156)

This PR converts the noteblocks for the 'web/api' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 15.

queengooborg committed:
Convert noteblocks for web/css folder (part 1) (#35157)

This PR converts the noteblocks for the 'web/css' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 1.

queengooborg committed:
Convert noteblocks for web/css folder (part 2) (#35158)

This PR converts the noteblocks for the 'web/css' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 2.

queengooborg committed:
Convert noteblocks for web/css folder (part 3) (#35159)

This PR converts the noteblocks for the 'web/css' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 3.

queengooborg committed:
Convert noteblocks for web/css folder (part 4) (#35160)

This PR converts the noteblocks for the 'web/css' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js). This is part 4.

PassionPenguin committed:
chore: convert broken noteblocks (#35151)

* chores: convert noteblocks

* Some more

* One more

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

Josh-Cena committed:
Fix macro redirection flaws (#35166)

Josh-Cena committed:
Capitalize first sentence of admonitions (#35163)

Josh-Cena committed:
Restore description of headers modification restrictions (#35165)

* Restore description of headers guards

* Rewrite

* Fix

* Move description

Josh-Cena committed:
Fix self-links & HTTP links (#35171)

OnkarRuikar committed:
Synchronize with BCD v5.5.42 (#35175)

* Synchronize with BCD v5.5.42

* Apply suggestions from code review

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

---------

Co-authored-by: OnkarRuikar <OnkarRuikar@users.noreply.github.com>
Co-authored-by: Joshua Chen <sidachen2003@gmail.com>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

mdn-bot committed:
Update InterfaceData based on WebRef (#35174)

OnkarRuikar committed:
fix(ci): add markdownlint custom rule for plain code fences (#35142)

* fix(ci): add markdownlint custom rule for plain code fences

* Update files/en-us/web/javascript/reference/global_objects/json/stringify/index.md

* Update .markdownlint.jsonc

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

Josh-Cena committed:
Ensure that all pages have a sidebar (#35177)

PassionPenguin committed:
chore: convert some noteblocks (#35180)

* feat: some noteblocks

* feat: convert noteblocks

OnkarRuikar committed:
fix(ci): add markdownlint custom rule for GFM blockquotes (#35178)

InSyncWithFoo committed:
Fix typos in `web/css/aspect-ratio` (#35181)

* Fix typos in `web/css/aspect-ratio`

* Apply suggestions from code review

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* Apply suggestion from review

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

DanKaplanSES committed:
Link to the Event Loop article (#35191)

wbamberg committed:
Fix CookieChangeEvent description (#35176)

in-in committed:
Fix typo (#35192)

janmarques committed:
Mention newline-escaping in template literal (#35193)

* Update index.md

* Update files/en-us/web/javascript/reference/template_literals/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

Josh-Cena committed:
Fix incorrectly rendered escaped braces (#35203)

Josh-Cena committed:
Fix broken glossary references (#35185)

* Fix broken glossary references

* Fix title

gagan-bhullar-tech committed:
cache-control erroneous request (#35199)

* cache control erroneous request

* Update files/en-us/web/http/headers/cache-control/index.md

* Update files/en-us/web/http/headers/cache-control/index.md

* response fix

* Update files/en-us/web/http/headers/cache-control/index.md

---------

Co-authored-by: Hamish Willee <hamishwillee@gmail.com>

OnkarRuikar committed:
fix typos (#35209)

* fix typo

* fix more

* remove repeated words

dletorey committed:
34701 ship transition behavior ff release (#34870)

* removed the transition-behavior property from experimental features

* added the release note for transition-behavior

* Update files/en-us/mozilla/firefox/releases/129/index.md

Co-authored-by: Vadim Makeev <hi@pepelsbey.dev>

---------

Co-authored-by: Vadim Makeev <hi@pepelsbey.dev>

dependabot[bot] committed:
Bump husky from 9.1.2 to 9.1.3 (#35225)

dependabot[bot] committed:
Bump fast-xml-parser from 4.3.2 to 4.4.1 (#35231)

nathanjensen1 committed:
Advertise Firefox 113+ AVIF animation support (#35229)

* Advertise Firefox 113+ AVIF animation support

https://www.mozilla.org/en-US/firefox/113.0/releasenotes/

* Add MDN ff release note and minor layout stuff

---------

Co-authored-by: Hamish Willee <hamishwillee@gmail.com>

Josh-Cena committed:
Update redirected links, part5 (#35221)

davidjb committed:
Document gz MIME type on Windows and macOS (#35211)

* Document gz MIME type on Windows and macOS

* Update files/en-us/web/http/basics_of_http/mime_types/common_types/index.md

* Match .zip to .gz

* Remove post from text

---------

Co-authored-by: Hamish Willee <hamishwillee@gmail.com>

gagan-bhullar-tech committed:
fix: for...in loop initializers are not ignored (#35237)

ignore initializers in loop

Josh-Cena committed:
Update redirected links, part1 (#35216)

* Update redirected links, part1

* Apply suggestions from code review

hamishwillee committed:
PerformanceResourceTiming.contentType - add docs (#35235)

* PerformanceResourceTimeing.contentType - add docs

* Make it more clear that content type is not same as mime type of file in some cases

* Update files/en-us/web/api/performanceresourcetiming/contenttype/index.md

---------

Co-authored-by: sideshowbarker <mike@w3.org>

hamishwillee committed:
HTML Sanitizer API - delete (#35238)

rjray committed:
Fix grammatical error, line 715 (#35248)

dependabot[bot] committed:
Bump husky from 9.1.3 to 9.1.4 (#35243)

mcaskill committed:
Move and rename "Accessibility concerns" section in CSS page templates (#34984)

* Move and rename accessibility section

Changes:
- Adjust section description for CSS Function and Property templates to match Selector template.
- Remove "optional" notice in favour of closing notice.

* Update description of Accessibility section in CSS page templates

As per: https://github.com/mdn/content/pull/34984#issuecomment-2245962862

Josh-Cena committed:
Fix broken anchor links (#35214)

* Fix broken anchor links

* Update index.md

Co-authored-by: wbamberg <will@bootbonnet.ca>

---------

Co-authored-by: wbamberg <will@bootbonnet.ca>

amaralis committed:
Extend warning in Set-Cookie article (#34826)

* Add warning to set-cookie article

Extends the first warning of the article, also related to browser behavior, to alert readers to to a relatively obscure and hard to debug characteristic of the Set-Cookie response header.

Ties together the additions in PR 34798 (https://github.com/mdn/content/pull/34798), offering the most relevant links to better help developers understand and fix issues if they arise.

* Update files/en-us/web/http/headers/set-cookie/index.md

Remove extra whitespace

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

* Clarify that behavior applies to CORS requests

* Update files/en-us/web/http/headers/set-cookie/index.md

Co-authored-by: Francisco Amaral <guedesmagalhaes@gmail.com>

* Update files/en-us/web/http/headers/set-cookie/index.md

---------

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

OnkarRuikar committed:
fix(mathml): remove two obsolete values (#35263)

OnkarRuikar committed:
Synchronize with BCD v5.5.43 (#35256)

Co-authored-by: OnkarRuikar <OnkarRuikar@users.noreply.github.com>

Josh-Cena committed:
Update redirected links, part2 (#35217)

gagan-bhullar-tech committed:
link style pages (#35213)

* link style pages

* fix

* style attribute

* review comments

* remove spaces

* Update files/en-us/web/html/global_attributes/style/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* Update files/en-us/web/api/htmlelement/style/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

dragunovartem99 committed:
Glossary: classes and functions are block scoped too (#35223)

* Add missing content, insert subheading for clarity

I've added example for function declarations, they can also be block-scoped (in strict mode)

You can check this in node.js or browser, I am learning a popular Udemy course (JS), and encountered this topic. I think it might be a usefull addition

I would be happy to receive feedback!

* Add brief explanation and link to block scoping rules

* Update files/en-us/glossary/scope/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* Update index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

hamishwillee committed:
Fix up .zip MIME exception to windows only (#35234)

DanKaplanSES committed:
Add consistently formatted "see also" links to all CSS value pages (#35210)

* Add consistently formatted "see also" links to all CSS value pages

* Remove mention of the article's own link in the "See also" section

Evergard committed:
fix: Update timetable section to reflect html change (#35017)

majaha committed:
`setTimeout` IDs can in fact be reused (#35249)

Correct info on setTimeout ID reuse

Web browsers can and will reuse setTimeout IDs eventually, as there is only a finite number of them.

atascii committed:
More examples for using slice() (#35226)

* Update index.md

In this case, I extended with more detail the operation of slice using negative indices and the combination of positive and negative indices.

* Update index.md

Updated the text correction warnings

* Update index.md

* Update index.md

Updated suggestions

* Update index.md

* Update files/en-us/web/javascript/reference/global_objects/array/slice/index.md

* Update files/en-us/web/javascript/reference/global_objects/array/slice/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

PassionPenguin committed:
feat: improvements on glossary links and their labels (#35246)

* feat: improvements on glossary links

* Apply suggestions from code review

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

* fix: macro

* fix: macro

* fix: macro

* fix: format

* Update files/en-us/glossary/effective_connection_type/index.md

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

* fix: macro

---------

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump snok/install-poetry from 1.3 to 1.4 (#35268)

Bumps [snok/install-poetry](https://github.com/snok/install-poetry) from 1.3 to 1.4.
- [Release notes](https://github.com/snok/install-poetry/releases)
- [Commits](https://github.com/snok/install-poetry/compare/v1.3...v1.4)

---
updated-dependencies:
- dependency-name: snok/install-poetry
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

mb21 committed:
What is JavaScript: link to local testing server (#34954)

* What is JavaScript: link to local testing server

* Update index.md

* Update files/en-us/learn/javascript/first_steps/what_is_javascript/index.md

Co-authored-by: Mauro Bieg <mb21@users.noreply.github.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

zarahzachz committed:
Fix typo in content categories page (#35245)

vic-ma committed:
Fix wording in "Inline formatting context" (#35250)

* Fix wording

* Update index.md

* Update files/en-us/web/css/inline_formatting_context/index.md

* Update files/en-us/web/css/inline_formatting_context/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

dawei-wang committed:
a11y(aria): Remove dead link (#35254)

* a11y(aria): Remove dead link

Fix mdn/content#35247

* Fix linting

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

* Update files/en-us/web/accessibility/aria/attributes/aria-label/index.md

---------

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

vic-ma committed:
Mention integer vs decimal difference (#35251)

* Mention integer vs decimal difference

* Update files/en-us/web/api/css_object_model/determining_the_dimensions_of_elements/index.md

* Update files/en-us/web/api/css_object_model/determining_the_dimensions_of_elements/index.md

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

hamishwillee committed:
Express tute - mongodb setup screens (#35206)

romanstetsyk committed:
fix: explicitly use `<i>` tag for italic text (#34886)

* fix: explicitly use `<i>` tag for italic text

* docs: add live samples to compare <i> and <em> tags

* Update files/en-us/web/html/element/em/index.md

* Apply suggestions from code review

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

eedrxs committed:
Fix typo (#35274)

Corrected grammatical error

wbamberg committed:
Fix 28752 part 4 (#35168)

* Fix issue 28752, part 4: update id and class attribute pages

* Move a para

* Recommend that id and class attributes should be valid CSS identifiers

* Update files/en-us/web/html/global_attributes/class/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* Update files/en-us/web/html/global_attributes/id/index.md

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

hamishwillee committed:
RTCRemoteInboundRtpStreamStats - plus alignment with outbound (#32657)

* RTCRemoteInboundRtpStreamStats: id

* RTCRemoteInboundRtpStreamStats.timestamp

* RTCRemoteInboundRtpStreamStats.type

* RTCRemoteInboundRtpStreamStats.transportId

* RTCRemoteOuboundRtpStreamStats.transportId - match to inbound

* RTCRemoteInboundRtpStreamStats.ssrc

* RTCRemoteOutboundRtpStreamStats.ssrc - alignment

* RTCRemoteInboundRtpStreamStats.kind

* RTCRemoteOutboundRtpStreamStats.kind - alignment

* Tweaks/aligment for transport id

* RTCRemoteInboundRtpStreamStats.codecId

* RTCRemoteOutboundRtpStreamStats.codecId - alignment

* RTCRemoteOutboundRtpStreamStats improve description

* RTCRemoteInboundRtpStreamStats - partial draft

* RTCRemoteInboundRtpStreamStats.jitter and glossary

* RTCRemoteInboundRtpStreamStats.packetsLost

* RTCRemoteInboundRtpStreamStats.packetsReceived

* Minor fixes to timestamp

* rttmeasurements property

* Add RTCRemoteInboundStats to sidebar

* Assorted minor improvements

* RTCRemoteInboutRtpStreamStats.totalrtt, localid, rttmeas

* RTCRemoteOutboundRtpStreamStats - matching localid, rttmeas, totalrtt

* RTCRemoteInboundRtpStreamStats.roundTripTime

* RTCRemoteInboundRtpStreamStats.fractionLost

* Apply suggestions from code review

Co-authored-by: wbamberg <will@bootbonnet.ca>

* Update files/en-us/web/api/rtcremoteinboundrtpstreamstats/index.md

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

* Stat properties in alphabetic order. Fix NOTE to new form

* fractionLost - tidy and point solely to algorithm for info on how it is calc.

* Note that the value is not precise, but is a  useful indicator

* Apply suggestions from code review

Co-authored-by: wbamberg <will@bootbonnet.ca>

---------

Co-authored-by: wbamberg <will@bootbonnet.ca>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

hamishwillee committed:
CSSPageRule.style update to CSSPageDescriptors (#34960)

* CSSPageDescriptors and related fixes

* Apply suggestions from code review

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

* Reorder params and remove link that will never exist

* use the new note format

---------

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

hamishwillee committed:
FF129 Relnote - PerformanceResourceTiming contentType + resourceStatus (#35236)

* FF129 Relnote - PerformanceResourceTiming contentType + resourceStatus

* Update files/en-us/mozilla/firefox/releases/129/index.md

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

---------

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

Josh-Cena committed:
Update redirected links, part3 (#35219)

* Update redirected links, part3

* Apply suggestions from code review

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

---------

Co-authored-by: Brian Thomas Smith <brian@smith.berlin>

meyerweb committed:
Added CSS shape-rendering property (#35228)

meyerweb committed:
Added CSS text-anchor property (#35167)

* Added text-anchor property

* Removed redirect

* Fixes for Josh-Cena

OnkarRuikar committed:
fix(css): update color-mix() page (#34371)

* fix(css): update color-mix() page

* add color-mixer tool

* apply review comments

* use macro

* small grammar update

---------

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

OnkarRuikar committed:
css(fix): update :read-only intro (#35060)

* css(fix): update :read-only intro

* fix live sample height

* Apply suggestions from code review

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* mention :not(:read-write)

* Apply suggestions from code review

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>

* Apply suggestions from code review

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

---------

Co-authored-by: Joshua Chen <sidachen2003@gmail.com>
Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

Josh-Cena committed:
Update redirected links, part7 (#35273)

estelle committed:
CSS geometric properties: add SVG note to height and width (#34934)

* CSS geometric properties: add SVG note to height and width

* Update files/en-us/web/css/width/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

---------

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

Djuffin committed:
Documentation for VideoFrameCopyToOptions' format and colorSpace (#35289)

gagan-bhullar-tech committed:
fix: trusted type api bogus example (#35252)

* trusted example

* review comments

* trusted pages fix

OnkarRuikar committed:
fix(css): font-stretch has been renamed to font-width (#35267)

* fix(css): font-stretch has been renamed to font-width.

* update note and convert gh livesample

* use font from shared-assets

* Apply suggestions from code review

Co-authored-by: wbamberg <will@bootbonnet.ca>

---------

Co-authored-by: wbamberg <will@bootbonnet.ca>

mdn-bot committed:
Update InterfaceData based on WebRef (#35290)

aniederer-chatham committed:
Correct when will-change forms a containing block, and simplify the l… (#35244)

* Correct when will-change forms a containing block, and simplify the list for absolute elements

From CSS Will Change Module Level 1:

If any non-initial value of a property would cause the element to generate a containing block for absolutely positioned elements, specifying that property in will-change must cause the element to generate a containing block for absolutely positioned elements.

If any non-initial value of a property would cause the element to generate a containing block for fixed positioned elements, specifying that property in will-change must cause the element to generate a containing block for fixed positioned elements.

I have personally tested this behavior in Chrome, Firefox, and Safari, and they all comply.

* fix typo

* `filter`, `transform` -> `filter` or `transform`

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

* Add period

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

* Add period

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

* Remove comma; add period

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

---------

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

OnkarRuikar committed:
css(fix): mention forward animations keep new staking context (#35056)

* css(fix): mention forward animations keep new staking context

* Apply suggestions from code review

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

* add the info to other pages

* Update files/en-us/web/css/animation/index.md

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

* review comments

---------

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

OnkarRuikar committed:
fix(css): update path() and shape-outside pages (#34961)

* fix(css): update path() and shape-outside pages

* convert the Github example to live sample

* tune easing

* Apply suggestions from code review

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

* add shape box with basic shape example in syntax section

* use new note syntax

---------

Co-authored-by: Chris Mills <chrisdavidmills@gmail.com>

Josh-Cena committed:
Fix unrendered Markdown (#35293)

gagan-bhullar-tech committed:
fix sgml link (#35301)

blm768 committed:
Improve clarity of setCustomValidity example (#35186)

Josh-Cena committed:
Remove hatch, hatchpath, color-profile from SVGData (#35172)

Josh-Cena committed:
Update redirected links, part6 (#35222)

rebloor committed:
runtime.getURL fixes (#35136)

* runtime.getURL fixes

* Feedback update

hamishwillee committed:
FF129 Relnote: HTTPS DNS records resolve using OS DNS resolver (#35215)

* FF129 Relnote: HTTPS DNS records resolve using OS DNS resolver

* Update files/en-us/mozilla/firefox/releases/129/index.md

* Update files/en-us/mozilla/firefox/releases/129/index.md

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

---------

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

hamishwillee committed:
FF129 Relnote: RTCDTMFSender.canInsertDTMF supported (#34862)

* FF129 Relnote: RTCDTMFSender.canInsertDTMF supported

* Apply suggestions from code review

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

* Fix example heading level

* Examples fix following feedback

* Apply suggestions from code review

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

---------

Co-authored-by: Dipika Bhattacharya <dipika@foss-community.org>

patrickjh committed:
Replace 'elements' with 'entries' (#35303)

sarajw committed:
Remove 'experimental' status from revert-layer (#35300)

* Remove 'experimental' key from revert-layer

It doesn't appear to be experimental any more :)

* Removing reference to compatibility table

Josh-Cena committed:
Fix bad DL (#35188)

OnkarRuikar committed:
Synchronize with BCD v5.5.44 (#35291)

* Synchronize with BCD v5.5.44

* remove unimplemented properties

---------

Co-authored-by: OnkarRuikar <OnkarRuikar@users.noreply.github.com>

bsmth committed:
feat(http): Replace usage of 'payload' with 'content' where appropriate

bsmth committed:
chore(http): update from main

bsmth commented:
oh no, sorry for notification spam, all 🙏🏻 I hit some nasty merge conflicts and resolved improperly. Recreating from a clean branch from main.
